train_prototype.py

```diff
@@ -140,6 +162,7 @@ def train(tasks, model, args, device):
     for key, value in vars(args).items():
         print(key + ' : ' + str(value))
     device = get_pytorch_device(args)
+    print(device)
```

Comment:
I think this line was to check that your laptop was using the GPU.

Reply:
We leave that for now I guess and can delete it later on once the device problem is fixed